pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -88,21 +85,60 @@ class InvalidSelectionError(DNFManagerError):
     """The software selection couldn't be resolved."""
 
 
+class DNFConfigWrapper(object):
+    """This is a temporary wrapper of a DNF config object."""
+
+    def __init__(self, config):
+        """Wrap the DNF config object."""
+        self._config = config
+
+    def __getattr__(self, name):
+        """Get the attribute.
+
+        Called when an attribute lookup has not found
+        the attribute in the usual places.
+        """
+        option = getattr(self._config, name)()
+        return option.get_value()
+
+    def __setattr__(self, name, value):
+        """Set the attribute.
+
+        Called when an attribute assignment is attempted.
+        """
+        if name in ["_config"]:
+            return super().__setattr__(name, value)
+
+        option = getattr(self._config, name)()
+        option.set(value)
```

Comment:
Regarding the `DNFConfigWrapper`, right now, it is quite ugly, since the option names are in the form `get_<name>_option`. I have two possible solutions:

1. Remove the wrapper and access the options directly. This would look like this:
    ```
    config.get_cachedir_option().get_value()
    config.get_cachedir_option().set(DNF_CACHE_DIR)
    ```

2. Keep the wrapper, but use simple `<name>` form in anaconda and wrap it with `"get_" + name + "_option"` in the `DNFConfigWrapper`.

The second solution would mean more readable access to the options, but the disadvantage is that it relies on a specific naming scheme in libdnf5. This could become a problem if we needed something else from the config e.g. to load config from parser (`ConfigMain::load_from_parser`), but it's not needed now. I prefer the second solution, but I don't know why the `DNFConfigWrapper` is described as temporary and what was the plan for future.

Reply:
I changed the wrapper to contain all the necessary options as properties. It got quite long, so let me know if it's acceptable. Maybe it could be moved to a separate file? Alternatively, I will change it to remove the wrapper entirely.